kani-compiler/src/kani_middle/transform/mod.rs

```diff
@@ -64,6 +66,7 @@ impl BodyTransformation {
         // This has to come after stubs since we want this to replace the stubbed body.
         transformer.add_pass(queries, AnyModifiesPass::new(tcx, &unit));
         transformer.add_pass(queries, ValidValuePass { check_type: check_type.clone() });
+        transformer.add_pass(queries, UninitPass { check_type: check_type.clone() });
```

Comment:
does the order matter?

Reply:
I have not tested it directly, but I would be inclined to place `UninitPass` after `ValidValuePass`. Right now, `UninitPass` only instruments read and writes via raw pointers, and I don't think that `ValidValuePass` emits any code that uses raw pointers. Vice-versa, I could see `ValidValuePass` emitting additional checks in the code generated by `UninitPass`.